docker-images/build.sh

```diff
@@ -6,7 +6,7 @@ source $(dirname $(realpath $0))/../tools/multi-platform-support.sh
 
 # Image directories
 base_images="base"
-java_images="operator jmxtrans"
+java_images="operator jmxtrans crd-convert"
```

Comment:
Again, these seems to be no image ... but it is still being added here.

Reply:
Removed ...